modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelToDDMFormConverter.java

```diff
@@ -114,7 +114,7 @@ protected DDMForm getConfigurationDDMForm() {
 		getConfigurationFieldOptionsProvider(
 			AttributeDefinition attributeDefinition) {
 
-		String pid = _configurationModel.getID();
+		String pid = _configurationModel.getBaseID();
```

Comment:
Hey @daiazius, I think, it will be safer if you do this only for the scoped configurations. What do you think?

Reply:
I was doing this at first, but after some testing I realized that doing a check for scoped then returning the base ID would be the same as just returning the base ID directly, as the ID when a configuration is not scoped is the base ID.

Reply:
thanks for checking